src/containers/BooksForm.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+import categories from '../sample-data/categories';
+
+class BooksForm extends React.Component {
+  state = {
+    title: '',
+    category: '',
+  };
+
+  handleChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { createBook: createBookDispatch } = this.props;
```

Comment:
I gotta ask why we need to rename `createBook` into `createBookDispatch` ? :D

Reply:
Good question. I had to rename it because of ESLint suggestions. It said that `createBook` was defined in outer scope. And indeed it was - import `createBook` action at top.
But here we receive another `createBook` from props, connected to redux (bound with dispatch method). 
I had to do the same in the `BooksList component`, I renamed `removeBook` and `changeFilter` methods.

Though I am not sure about this approach. What do you think?

Reply:
Another approach would be:
- import actions like this:
`import * as actions from '../actions';`

- and pass the `actions` object into the `connect` method.
```javascript
export default connect(mapSateToProps, actions)(Components);
```
But in this case it will be passing all the actions to the components, even though I need only one action in `BooksForm` component and two actions in `BooksList`.

Reply:
That means you will be importing all the other actions too, even if they're not used for this component. :) I'm not saying that we should always override ESLint's rules but in this case I think we can just include one in eslintrc to exclude this particular error.